interfaces/Portalicious/src/app/domains/payment/payment.model.ts

```diff
@@ -2,6 +2,8 @@ import { PaymentReturnDto } from '@121-service/src/payments/transactions/dto/get
 
 import { Dto } from '~/utils/dto-type';
 
+import { ProgramPaymentsStatusDto } from '../../../../../../services/121-service/src/payments/dto/program-payments-status.dto';
```

Comment:
Should be non-relative import (I also added a [task to make sure eslint complains about this](https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/30934)).